Lombiq.ChartJs.Tests.UI/Extensions/UITestContextExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using Lombiq.ChartJs.Samples.Controllers;
+using Lombiq.HelpfulLibraries.OrchardCore.Mvc;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using System.Drawing;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Lombiq.ChartJs.Tests.UI.Extensions;
+
+public static class UITestContextExtensions
+{
+    public static Task ExecuteChartJsSampleRecipeDirectlyAsync(this UITestContext context) =>
+        context.ExecuteRecipeDirectlyAsync("Lombiq.ChartJs.Samples.Content");
+
+    public static Task GoToBalanceAsync(this UITestContext context) =>
+        context.GoToAsync<SampleController>(controller => controller.Balance());
+
+    public static Task GoToHistoryAsync(this UITestContext context) =>
+        context.GoToAsync<SampleController>(controller => controller.History(null, null));
+
+    /// <summary>
+    /// Takes a screenshot of an element.
+    /// </summary>
+    public static Bitmap TakeScreenshotImage(this UITestContext context, IWebElement element)
```

Comment:
Delete this, use the one you made [here](https://github.com/Lombiq/UI-Testing-Toolbox/pull/158#discussion_r864744747). (I see the latest OSOCE commit doesn't reference that submodule. Did you forget to push some commits?)

Reply:
I'll move it in a next step. OSOE-111